pytorch_lightning/loggers/wandb.py

```diff
@@ -356,16 +357,16 @@ def watch(self, model: nn.Module, log: str = "gradients", log_freq: int = 100, l
 
     @rank_zero_only
     def log_hyperparams(self, params: Union[Dict[str, Any], Namespace]) -> None:
-        params = self._convert_params(params)
-        params = self._flatten_dict(params)
-        params = self._sanitize_callable_params(params)
+        params = _convert_params(params)
+        params = _flatten_dict(params)
```

Comment:
off-topic. But I think wandb supports logging nested dict for hparams. Maybe we shouldn't force the behavior here.